code/modules/error_handler/error_handler.dm

```diff
@@ -0,0 +1,95 @@
+/var/list/error_last_seen = list()
+// error_cooldown items will either be positive (cooldown time) or negative (silenced error)
+//  If negative, starts at -1, and goes down by 1 each time that error gets skipped
+/var/list/error_cooldown = list()
+/var/total_runtimes = 0
+/var/total_runtimes_skipped = 0
+// The ifdef needs to be down here, since the error viewer references total_runtimes
+#ifdef DEBUG
+/world/Error(var/exception/e)
+	if (!istype(e)) // Something threw an unusual exception
+		world.log << "\[[time_stamp()]] Uncaught exception: [e]"
+		return ..()
+
+	if (!global.error_last_seen) // A runtime is occurring too early in start-up initialization
```

Comment:
In that case, there's a different pre-config point of failure you'll need to worry about: when an error makes its way to the error viewer, and it tries to notify admins with log_debug(), it'll runtime due to log_debug checking config blindly.